containers/Makefile

```diff
@@ -194,7 +200,7 @@ build/repositories/concent: build/package-builder/clone-or-update.sh /var/concen
 build/repositories/golem/: build/package-builder/clone-or-update.sh /var/concent-data/golem
 	build/package-builder/clone-or-update.sh  "golem" "$@"
 
-build/%/middleman_protocol/: build/repositories/concent
+build/%/middleman_protocol/: build/repositories/concent build/virtualenv/
 	cp --recursive "$</middleman_protocol/." "$@"
```

Comment:
Why does `middleman_protocol/` require having a virtualenv with Concent's requirements on the host?

You're only making a copy of middleman's source. You're not running it. This should not require installing any dependencies. If any other target that depends on `build/%/middleman_protocol/` needs this virtualenv then it should be the dependency of that target.

Reply:
The virtualenv is uses here for unification purpose. This create middleman package already. The problem is because some containers are copy this directory before running virtualenv and some of it after. The solution is maybe create docker `nginx-proxy` image which is uses `virtualenv` after all others.

Reply:
Let's talk about it tomorrow. I need more details.

Reply:
Conclusions from our discussion:
- You do this because after the installation of Python packages in the virtualenv an `.egg-info` directory gets created inside `middleman_protocol/` and you want to have it in the Docker image too for consistency.
    - We don't need this `.egg-info`. It gets created because `middleman-protocol` gets installed with `pip --editable`.
    - Let's switch to installing it with `setup.py install`
    - Let's install it with `setup.py install` on the host too. This means you should use `cacheable-requirements` on the host too.
    - After installing `middleman-protocol` in the Docker image with `setup.py install`, remove the `middleman_protocol/` directory you installed from. It won't be used at runtime and having it there would be misleading.